src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -80,21 +81,21 @@ if (__DEV__) {
 
 }
 
+function handleElement(element) {
+  if (typeof element.type !== 'string') {
+    return;
+  }
+  for (var key in element.props) {
+    warnUnknownProperty(key, element._source);
+  }
+}
+
 var ReactDOMUnknownPropertyDevtool = {
-  onCreateMarkupForProperty(name, value) {
-    warnUnknownProperty(name);
-  },
-  onSetValueForProperty(node, name, value) {
-    warnUnknownProperty(name);
```

Comment:
I believe this fixed the issue because the prop names were being filtered prior to this event firing, which means we only fire this event on props that are "correct".  This is a reasonable behavior, because some devtools (eg. perf) might want to know about the DOM operations that are actually being performed, rather than every prop that was specified.  We can re-think how/when those events fire, but that's a much longer discussion.